homework5/task2.js

```diff
@@ -0,0 +1,27 @@
+function WorkingPersonFunc(object) {
+  PersonFunc.apply(this, arguments);
+}
+
+WorkingPersonFunc.prototype = Object.create(PersonFunc.prototype);
+WorkingPersonFunc.prototype.constructor = WorkingPersonFunc;
+
+person1 = new WorkingPersonFunc(SarahMay);
+person2 = new WorkingPersonFunc(JeremieBrown);
+person3 = new WorkingPersonFunc(Elizabeth);
```

Comment:
SarahMay -> undefined
JeremieBrown -> undefined
Elizabeth -> undefined
These objects should be inited in the current file